benchmark/single-source/SortLargeExistentials.swift

```diff
@@ -0,0 +1,81 @@
+//===--- SortLettersInPlace.swift -----------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// This test is a variant of the SortLettersInPlace.
+
+import TestsUtils
+
+protocol LetterKind {
+  var value: String { get }
+  func lessthan(_ rhs: LetterKind) -> Bool
+}
+
+// A struct which exeeds the size of the existential inline buffer.
+struct Letter : LetterKind {
+  let value: String
+
+  var a: Int = 27
+  var b: Int = 27
+  var c: Int = 27
+  var d: Int = 27
```

Comment:
to make the struct larger than 3 words to ensure that it does not fit into the 3-word existential inline buffer.